CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ x.x.x Release notes (yyyy-MM-dd)
   example, users who are authenticated using third-party credentials will find
   themselves logged out of the Realm Object Server if the third-party identity
   service indicates that their credential is no longer valid.
+* Search depth when reporting changes to observation blocks has been
```

Comment:
I'd suggest rephrasing this to focus on the symptom that's being fixed, and to note that it's related to collection notifications. Perhaps:

> Fix high CPU usage and hangs when processing collection notifications in highly-connected object graphs.

I'm not sure that it's worth mentioning the change to maximum depth?